app/components/public/side-menu.js

```diff
@@ -3,12 +3,18 @@ import { computed } from '@ember/object';
 import moment from 'moment';
 
 export default Component.extend({
+  selectedIndex: undefined,
```

Comment:
Can you make it `index`?

Reply:
We can do this but having  
```
actions: {
    changeIndex(index) {
      this.set('Index', index);
    }
  }
```
will be very confusing for future issues, if any.